### PR TITLE
Create messages.yml - add Ukrainian locale

### PR DESCRIPTION
**Pull request by perdolka**

### CODE REVIEW

**Comment by gera2ld** [comment]
The leading spaces in each line are essential, otherwise the yaml file gets broken. 😅

**Comment by perdolka** [comment]
Fixed.

File: src/_locales/uk/messages.yml [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+  message: У вас ще не встановлено жодного скрипта.
+labelNoSearchScripts:
+  description: Message shown when no script is found in search results.
+  message: Скрипти не знайдено.
+labelNotifyUpdates:
+  description: Option to show notification when script is updated.
+  message: Повідомляти про оновлення скриптів.
+labelRelated:
+  description: Label of related links.
+  message: 'Пов'язані посилання: '
```
Comment (gera2ld) [inline src/_locales/uk/messages.yml]: Syntax error, should be `'Пов''язані посилання: '`.

**Comment by perdolka** [comment]
Fixed.

**Pull request merged**